### ISSUE TITLE
Prolonged central motor conduction time

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7768]
**Preferred term label:**
Prolonged central motor conduction time

**Synonyms**

**Definition (free text, please give PubMed ID)**
Increased duration of the central motor conduction time (CMCT). The CMCT estimates the conduction time of corticospinal fibers between motor cortex and spinal (or bulbar) motoneurons. It includes the times for excitation of cortical cells, conduction via the corticobulbar-corticospinal tract and excitation of the motoneuron sufficient to reach its firing threshold. CMCT is calculated by subtracting the peripheral conduction time from the MEP latency recorded after cortical stimulation.

The determination of central motor conduction time (CMCT) is a sensitive method to detect myelopathies, and abnormalities may be observed in the absence of radiological changes.[3](https://www.nature.com/articles/sc201448#ref-CR3)
PMID: 24752292

**Parent term (use hpo.jax.org/app)**
Abnormality of central motor conduction HP:0012079

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
616282 
PMID:23973755   6/6

**Comment by pnrobinson** [comment]
added
